src/translations/en.json

```diff
@@ -3922,6 +3922,11 @@
               "certificate_expire": "Certificate renewal at {date}",
               "more_info": "More details"
             },
+            "ice_servers": {
+              "title": "Cloud WebRTC ICE servers",
+              "info": "Do you want to use Home Assistant Cloud for WebRTC connections? Enable this option to use the Home Assistant Cloud WebRTC ICE servers.",
+              "link_learn_how_it_works": "Learn how it works"
+            },
```

Comment:
These are yet to change as well.

Reply:
When will that be ready?

Reply:
This is now changed to the correct texts.